omnibus/package-scripts/installer-deb/preinst

```diff
@@ -3,7 +3,4 @@
 #
 # .deb: STEP 2 of 5
 
-
-SYSTEMCTL_SKIP_SYSV=true systemctl stop datadog-installer || true
```

Comment:
This doesn't seem related to the agent itself nor its users, could you clarify why it's needed?

Reply:
We aren't currently installing the systemd unit for the installer and this will likely be done inside of the installer instead. 
We could leave it but given that we've hit a few issues with the QA already I want to trim down anything that could be an issue for step1

Reply:
Fair enough, thanks for clarifying :)